typescript/schemas/src/internal/generatePyclass.ts

```diff
@@ -594,7 +609,7 @@ impl ${channelClass} {
     }
 
     fn __repr__(&self) -> String {
-        format!("${channelClass}(topic='{}')", self.0.topic()).to_string()
+        format!("${channelClass}(id='{}',topic='{}')", self.id(), self.topic()).to_string()
```

Comment:
The `, ` separator is preferred over `,`; and I think we can drop the quotes around ID (as above).

```suggestion
        format!("${channelClass}(id={}, topic='{}')", self.id(), self.topic()).to_string()
```